File: docs/components_page/components/tooltip/tooltip.py
import dash_bootstrap_components as dbc
import dash_html_components as html

tooltip = html.Div(
    [
        html.P(
            [
                "I wonder what ",
                html.Span(
                    "floccinaucinihilipilification",
                    id="tooltip-target",
                    style={"textDecoration": "underline", "cursor": "pointer"},
                ),
                " means?",
            ]
        ),
        dbc.Tooltip(
            "Noun: rare, "
            "the action or habit of estimating something as worthless.",
            target="tooltip-target",
        ),
    ]
)

File: docs/components_page/components/tooltip/tooltip.R
library(dashBootstrapComponents)

tooltip <- div(
  list(
    p(
      list(
        "I wonder what ",
        span(
          "floccinaucinihilipilification",
          id = "tooltip-target",
          style = list(textDecoration = "underline", cursor = "pointer")
        ),
        " means?"
      )
    ),
    dbcTooltip(
      paste(
        "Noun: rare,",
        "the action or habit of estimating something as worthless."
      ),
      target = "tooltip-target"
    )
  )
)
